### ISSUE TITLE
In scan mode, Narrator does not announce 'Selected' while navigating on ChoiceGroup using enter (Keyboard)

### ISSUE THREAD

**Issue by Sneha-Mindtree** [microsoft/fluentui#13787]
In scan mode, we can use down arrow or up arrow to navigate towards the ChoiceGroup (radio buttons) and then press enter to select an unselected choice group, Narrator does not read as selected. Narrator does not announce 'Selected' while navigating on Radio button.
### Environment Information
OS: Win 10 (2004)
Build: OS Build (19613.1005)
Browser: Edge Chromium DEV Build (84.0.522.5) , Firefox(76.0)
ScreenReader: Narrator, NVDA

Issue also exists with Firefox + NVDA

Repro Steps
Open https://developer.microsoft.com/en-us/fluentui#/controls/web/choicegroup in Chrome/Firefox and turn on Narrator.
Navigate to Usage - Basic ChoiceGroup
Switch narrator to Scan mode (Caps + Space).
Navigate to any unselected choice group by using down arrow keys...press enter and verify the issue.

reproduction of the bug in a codepen :
Go to https://codepen.io/sneha-Mindtree/pen/WNrERZL
Switch narrator to Scan mode (Caps + Space).
Navigate to any unselected choice group by using down arrow keys...press enter and verify the issue.

#### Actual behavior
In scan mode, Narrator does not announce 'Selected' while navigating using enter on ChoiceGroup. 

#### Expected behavior:
In scan mode, Narrator should announce 'Selected' while navigating using enter on ChoiceGroup. 

### Priorities 
Products/sites affected: AI Builder project
